risc0/build_kernel/src/lib.rs

```diff
@@ -141,7 +141,9 @@ impl KernelBuild {
 
         let mut flags = vec![];
         if let Ok(nvcc_flags) = env::var("RISC0_NVCC_FLAGS") {
-            flags.push(nvcc_flags);
+            for flag in nvcc_flags.split(" ") {
+                flags.push(flag.to_string());
+            }
```

Comment:
This won't work with things like `RISC0_NVCC_FLAGS=--foo='x y'`. This issue might not be worth worrying about for this use case, but this could solve it: https://docs.rs/shlex/latest/shlex/fn.split.html

Reply:
thanks for catching this!